virttest/utils_test/__init__.py

```diff
@@ -1902,8 +1902,9 @@ def __init__(self, stress_type, params, download_url="", make_cmds="",
         self.uninstall_cmds = self.uninstall_cmds or './configure && make uninstall'
         self.work_path = self.params.get('%s_work_path' % stress_type,
                                          work_path)
-        self.check_cmd = "pidof -s %s" % stress_type
-        self.stop_cmd = "pkill -9 %s" % stress_type
+        check_cmd = self.params.get('stress_cmds_%s' % stress_type, stress_cmds).split(" ")[0]
```

Comment:
Maybe use `self.stress_cmds` as the default is better, as the `launch_cmds` comes form   `lauch_cmds = 'nohup %s %s &' % (self.stress_cmds, self.stress_args)`

Reply:
Agree with @hereischen,  `stress_cmds` could be empty, when user define it in `params`. `self.stress_cmds` will cover all the cases.

Reply:
yeah , its same. I can have it as
```
check_cmd = self.stress_cmds.split(" ")[0]
```